### PR TITLE
First commits

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request contains the first commits for this Terraform module.

## 💭 Motivation and context ##

We want to create an SNS topic in each AWS account that can be used to send a message to the account email when a CloudWatch alarm is triggered.

## 🧪 Testing ##

All automated tests pass, and I have deployed this module to the SharedServices account in our staging COOL environment via cisagov/cool-accounts#118.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: locals.tf [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# ------------------------------------------------------------------------------
+# We can get the account ID of this account from the provider's caller
+# identity.
+# ------------------------------------------------------------------------------
+data "aws_caller_identity" "this" {
+}
+
+# Retrieve the information for all accounts in the organization.  This
+# is used, for instance, to lookup the account IDs for the user
+# account.
```
Comment (dav3r) [inline locals.tf]: This comment should be updated to reflect the current use case:
```suggestion
# Retrieve the information for all accounts in the organization.
# This is used to look up the email address associated with the
# calling account.
```

**Review by dav3r — APPROVED** [review]
Looks great, I just noted one comment that could use a fresh coat of paint.

File: README.md [hunk 2]
```diff
@@ -51,42 +45,30 @@ No modules.
...
-| id | The EC2 instance ID. |
-| private\_ip | The private IP of the EC2 instance. |
-| subnet\_id | The ID of the subnet where the EC2 instance is deployed. |
+| sns\_topic | The SNS topic to which a message is sent when a CloudWatch alarm is triggered. |
 
 ## Notes ##
 
 Running `pre-commit` requires running `terraform init` in every directory that
 contains Terraform code. In this repository, these are the main directory and
 every directory under `examples/`.
```
Comment (mcdonnnj) [inline README.md]:
```suggestion
contains Terraform code. In this repository,  this is only the main directory.
```

**Review by mcdonnnj — APPROVED** [review]
Makes sense. I had one suggestion given the removal of the `examples/` directory.

**Pull request merged**